py3status/modules/getjson.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+"""
+Query a url and display the json response
+
+Configuration parameters:
+    cache_timeout: how often we refresh this module in seconds
+        (default 30)
+    format: placeholders will be replaced by the returned json key names
+        (default '')
+    negative_cache_timeout: how often to check again when offline
+        (default 2)
```

Comment:
I'd remove this.  I know that whatsmyip uses this but really going forward we should be looking at a better way for modules to refresh if the internet becomes available.